### PR TITLE
Update support data for touch-action property

### PR DESCRIPTION
**Pull request by huijing**
Addresses #3325

### CODE REVIEW

**Comment by huijing** [comment]
There are 2 separate files for touch-action level 1 and touch-action level 2 but caniuse.com only shows the information for touch-action level 1. Is this because touch-action level 2 is still in WD status?

File: features-json/css-touch-action-2.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
       "8":"n",
       "9":"n",
       "10":"a x #1",
-      "11":"a #1"
+      "11":"n"
```
Comment (lpd-au) [inline features-json/css-touch-action-2.json]: This seems patently incorrect to me.

File: features-json/css-touch-action-2.json [hunk 2]
```diff
@@ -274,7 +274,7 @@
     },
     "ie_mob":{
       "10":"a x #1",
-      "11":"a #1"
+      "11":"n"
```
Comment (lpd-au) [inline features-json/css-touch-action-2.json]: Ditto for desktop IE

File: features-json/css-touch-action-2.json [hunk 3]
```diff
@@ -288,7 +288,7 @@
   },
   "notes":"Due to scoping issues of the W3C Pointer Events Working Group, `pinch-zoom` appears in the WHATWG Compatibility Standard instead of W3C Pointer Events Level 2.",
   "notes_by_num":{
-    "1":"IE and Edge only support `pinch-zoom`.",
+    "1":"Edge only supports `pinch-zoom`.",
```
Comment (lpd-au) [inline features-json/css-touch-action-2.json]: Ditto

File: features-json/css-touch-action.json [hunk 4]
```diff
@@ -47,7 +47,7 @@
       "8":"n",
       "9":"n",
       "10":"y x #2",
-      "11":"y"
+      "11":"n"
```
Comment (lpd-au) [inline features-json/css-touch-action.json]: IE10 was the first browser to support pointer events and touch-action, what gives you the impression it disappeared in IE11?

File: features-json/css-touch-action.json [hunk 5]
```diff
@@ -186,8 +186,8 @@
       "9":"n",
       "9.1":"a #3",
       "10":"a #3",
```
Comment (lpd-au) [inline features-json/css-touch-action.json]: 9.1 and 10 should match 10.1 and TP

File: features-json/css-touch-action.json [hunk 6]
```diff
@@ -284,7 +284,7 @@
     },
     "ie_mob":{
       "10":"y x #2",
-      "11":"y"
+      "11":"n"
```
Comment (lpd-au) [inline features-json/css-touch-action.json]: Ditto previous comments re IE

**Comment by huijing** [comment]
@lpd-au thank you for reviewing. You are absolutely right, I forgot that IE11 does NOT support feature queries to begin with so that was a false negative. I've updated the pull request accordingly, as well as for the Safari data.

**Comment by lpd-au** [comment]
LGTM now, thanks :)

Fyi, I added the file for the spec level 2 values in a pull request that's still labelled needs data review, I'm not 100% sure who's trusted to conduct that though. The spec status shouldn't matter, afaik it's all about data integrity.

**Comment by Fyrd** [comment]
@huijing Thanks!

@lpd-au I will review the data, eventually.

**Pull request merged**